### PR TITLE
[DEVOPS-61]  repair Appveyor caching

### PR DESCRIPTION
**Pull request by deepfire**
Reduces appveyor build times from 45min to 15min.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@ build: off
 
 environment:
   global:
+    VERSION: "1.0"
```
Comment (domenkozar) [inline appveyor.yml]: This should use `APPVEYOR_BUILD_VERSION` to reuse the first line

File: appveyor.yml [hunk 2]
```diff
@@ -36,7 +37,7 @@ cache:
   # different branch is built.
   - "%BRANCH_CACHE_DIR%"        # e.g. C:\cache\feature_abc123
   - "%CACHE_DIR%\\master"
-  - "%CACHE_DIR%\\cardano-sl-0.4"
+  - "%CACHE_DIR%\\cardano-sl-%VERSION%"
```
Comment (domenkozar) [inline appveyor.yml]: Nice one!

**Comment by deepfire** [comment]
@domenkozar, this fails, as per https://issues.serokell.io/issue/DEVOPS-61#comment=96-8219

**Comment by domenkozar** [comment]
Restarting fixed it.

File: appveyor.yml (hunk 2 above)
Comment (domenkozar) [reply appveyor.yml]: @deepfire https://www.appveyor.com/docs/environment-variables/

if it's not possible to merge the two variables, they should have corresponding comments to update both

Comment (deepfire) [reply appveyor.yml]: @domenkozar, done: https://ci.appveyor.com/project/jagajaga/cardano-sl/build/1.0.14818#L7

**Comment by deepfire** [comment]
@domenkozar, this PR is ready.

**Comment by domenkozar** [comment]
@deepfire it needs to be made against 1.0 branch

**Pull request merged**